### DISCUSSION TITLE
Add GUI to DOLPHYN

### DISCUSSION THREAD

**Discussion by RuaridhMacd** [macroenergy/DOLPHYN#140] [Ideas]
By making DOLPHYN easier to use, we will speed up our work, reduce mistakes and reach a broader audience of users. A graphical user interface (GUI) would be a big improvement in this respect. 

Some ideas for how this could be done:
- The GUI could have two parts:
    - A scenario generator / editor, where cases are created by dragging and dropping nodes, resources, demands, etc together.
    - An interface to organize and run multiple optimizations, perhaps changing some parameters or variables each time.

- The GUI could be developed in Julia or another language, but it might be interesting to make it browser-based so that the same codebase can be used more easily by users with different operating systems (though the same issues exist for different browsers.
    - This would also let us use many existing tools and solutions for producing interactive software.

**Comment by Betristor** [comment]
This is really a crazy one. But I loved it!
